ModulBot/Commands/CreateStat.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using ModulBot.Attributes;
+using ModulBot.Database.PostgresRepositories;
+using ModulBot.Interfaces;
+using ModulBot.Models;
+using Telegram.Bot.Types;
+
+namespace ModulBot.Commands
+{
+    [NotShowInHelp]
+    [Serializable]
+    public class CreateStat : ITool
+    {
+
+        public string Description { get; set; }
+        public List<string> CommandsName { get; set; }
+        public bool IsAdmin { get; set; }
+        private static int step = 0;
+
+        public async void Run(Message message)
+        {
+            if (message.Text[0] == '/')
+            {
+                message.Text = message.Text.Substring(1);
+            }
+
+            switch (step)
+            {
+                case 0:
+                    if (message.Chat.Id < 0) //в групповых чатах бот не видит обычных сообщений
+                    {
+                        await Bot.BotClient.SendTextMessageAsync(message.Chat.Id, "*Внимание!* В групповых" +
+                            " чатах перед ответом ставьте слэш \"*/*\", " +
+                            "т.к. бот не видит обычных сообщений.\nНапример: */да*, */1*, и т.д.",
+                            Telegram.Bot.Types.Enums.ParseMode.Markdown);
+                    }
+
+                    Bot.IsDialogStart = true;
+                    step++;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this set, which updates a 'static' field from an instance method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2696)